Refresh of all the 'Other' how-tos. (#381)
epinio/docs: docs/howtos/other/gitjob_push.md

there is a feature that lets you set up a deployment that rebuilds and republishes,
when your code repository in Git changes

You can have this functionality in Epinio using the GitJob CRD from [Rancher Fleet](https://fleet.rancher.io/).

## Setup

### Install GitJob

If you don't have Rancher, or stand-alone Fleet installed, you need to
[install the GitJob operator](https://github.com/rancher/gitjob#running).

Now you need to setup the Service Account to run your Jobs with.
You don't need to do anything using the Kube API, so you don't need to add any role bindings to it:

```yaml
apiVersion: v1
kind: ServiceAccount
metadata:
  name: epinio-gitjob
```

### Check Epinio installation

If you are using Rancher, you can install Epinio directly from there.
For more information check the
[Rancher configuration](../../installation/other_inst_scenarios/install_epinio_on_rancher.md) with Epinio.

Once you have deployed Epinio,
log in as admin and verify that the namespace `workspace` exists.
It also needs to be the current target.
You can do this, if necessary, by running:

```console
epinio namespace create workspace 
epinio target workspace 
```

### Upload Epinio settings

> So the GitJob can authenticate and push correctly, you can upload your Epinio settings file to the cluster with:

Reviewer: ```suggestion
For the GitJob to authenticate and push correctly, upload your Epinio settings file to the cluster with:
```
